docs/README.md

```diff
@@ -39,6 +39,7 @@
   * [Pausing the Test](articles/documentation/test-api/pausing-the-test.md)
   * [Handling Native Dialogs](articles/documentation/test-api/handling-native-dialogs.md)
   * [Working with &lt;iframes&gt;](articles/documentation/test-api/working-with-iframes.md)
+  * [A-Z](articles/documentation/test-api/a-z.md)
```

Comment:
We can surely name it `Index` but nothing will be more simple and eye-catching for an alphabetical list than `A-Z`.
This topic is aimed to provide fast reference, for example, when someone gets puzzled with someone else's test code. `A-Z` will stand out from other topics.